palantirnet/the-build
# Move Acquia DB backups to cloud v2.

https://palantir.atlassian.net/browse/DEV-32

## Test Intructions:

* Add this to an existing project and run `composer install`
* Ensure in the `build.yml` from the existing project you have the following variables:
   
   ```
  # Configuration for targets/acquia.xml
  acquia:
    backups: 'artifacts/backups'
    site: 'palantirnet'
    database: 'palantirnet'
    env: prod (default one if not defined)
   ```
   
  * Ensure there is a `.env` with acquia clould credentials specified in the 1Password
  
  * Run `ddev restart`
  
  * Run `ddev phing acquia-get-backup`
  
  * Ensure backup was downloaded inside the backups directory specified in `backups`
  
  * Run `ddev phing drupal-load-db` and ensure database dump downloaded is imported correctly.
